src/main/java/crawler/AnalyzePage.java

```diff
@@ -0,0 +1,24 @@
+package crawler;
+
+import crawler.queue.DownloadQueue;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+
+public class AnalyzePage {
+
+
+    public void analyze(Document doc, DownloadQueue downloadDb) throws IOException, InterruptedException {
+
+        Elements hrefs = doc.select("a[href]");
+        for (Element href : hrefs) {
+//            System.out.println(href.attr("abs:href"));
+            if ((href!=null)&&(href.attr("abs:href").contains(doc.baseUri()))) {
```

Comment:
1. formatting 
2. move string to a constant
3. some pairs of () are unnecessary